src/Core/Enums/ReferenceEventType.cs

```diff
@@ -39,5 +39,7 @@ public enum ReferenceEventType
     [EnumMember(Value = "collection-created")]
     CollectionCreated,
     [EnumMember(Value = "organization-edited-by-admin")]
-    OrganizationEditedByAdmin
+    OrganizationEditedByAdmin,
+    [EnumMember(Value = "organization-created-by-admin")]
+    OrganizationCreatedByAdmin
```

Comment:
Is this used anywhere?

Reply:
It will be used in https://bitwarden.atlassian.net/browse/EC-431. I'll move it to that PR